src/lib/i18n/texts/de.json

```diff
@@ -66,22 +66,22 @@
       "losses": "Niederlagen",
```

Comment:
if you look above this line it actually has byes filled in (its duplicated) - is that the right context?